server/src/userManagement.mjs

```diff
@@ -84,9 +84,13 @@ function getWssForUser(userId) {
   return undefined;
 }
 
+//get the ws for user,if user has ws already,it will return the latest ws connection
 function getWsForUser(userId) {
+  let wsArray = [];
```

Comment:
Nit: You don't need this here since you only use wsArray in the if block. Or if you do, you don't have to assign it.

Reply:
Moved the code inside if block,It is expecting me to assign it explicitly as an array.else i'm getting the below error.
 **TypeError: Cannot read properties of undefined (reading 'push')**